packages/angular/cli/models/architect-command.ts

```diff
@@ -5,12 +5,8 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
-import {
-  Architect,
-  BuilderConfiguration,
-  BuilderContext,
-  TargetSpecifier,
-} from '@angular-devkit/architect';
+import { index2 } from '@angular-devkit/architect';
```

Comment:
nit: maybe `import { index2 as architect }` so that later when we remove the old architect API we don't have so many delta lines where this is referenced below?

Reply:
The imports will be direct once we switch over completely so we’ll end up with the same delta.